ShopifySharp/Services/AssignedFulfillmentOrder/AssignedFulfillmentOrderService.cs

```diff
@@ -0,0 +1,24 @@
+﻿using ShopifySharp.Filters;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace ShopifySharp
+{
+    public class AssignedFulfillmentOrderService : ShopifyService, IAssignedFulfillmentOrderService
+    {
+        /// <summary>
+        /// Creates a new instance of <see cref="AssignedFulfillmentOrderService" />.
+        /// </summary>
+        /// <param name="myShopifyUrl">The shop's *.myshopify.com URL.</param>
+        /// <param name="shopAccessToken">An API access token for the shop.</param>
+        public AssignedFulfillmentOrderService(string myShopifyUrl, string shopAccessToken) : base(myShopifyUrl, shopAccessToken)
+        {
+        }
+
+        public virtual async Task<IEnumerable<FulfillmentOrder>> ListAsync(AssignedFulfillmentOrderFilter filter, CancellationToken cancellationToken = default)
```

Comment:
Should this return an `IEnumerable<AssignedFulfillmentOrder>` instead of `IEnumerable<FulfillmentOrder>`? If not, should we move this method into the `FulfillmentOrderService` and name it something like `ListAssignedFulfillmentOrdersAsync`?

Reply:
The AssignedFulfillmentOrder is an interesting one. In the source documentation, it is a separate service. In reality, it is similar to a GetAll endpoint in that it returns all fulfillment orders that your fulfillment service can see. Also, the source docs show that they are returning different properties. But in Postman, you can see that they are the same. It really is just a filtered GetAll.

Reply:
Is there still something that needs to be changed? I have a project that is waiting for this PR.

Reply:
Also would like to use this functionality for our app too! Waiting to see if it is merged. Cheers

Reply:
Also looking to use this, is it going to get merged in? @nozzlegear @codecooper